src/features/channels/components/channel-list/search-channel/SearchChannel.tsx

```diff
@@ -15,48 +22,109 @@ import ListItemIcon from '@mui/material/ListItemIcon';
 import ListItemText from '@mui/material/ListItemText';
 import FormControl from '@mui/material/FormControl';
 import IconButton from '@mui/material/IconButton/IconButton';
+import IndeterminateCheckBoxIcon from '@mui/icons-material/IndeterminateCheckBox';
 import Input from '@mui/material/Input';
 import InputAdornment from '@mui/material/InputAdornment';
 import InputLabel from '@mui/material/InputLabel';
 import MenuIcon from '@mui/icons-material/Menu';
 import MailIcon from '@mui/icons-material/Mail';
 import InboxIcon from '@mui/icons-material/MoveToInbox';
+import MarkEmailReadIcon from '@mui/icons-material/MarkEmailRead';
+import PersonAddIcon from '@mui/icons-material/PersonAdd';
+import PhotoCameraIcon from '@mui/icons-material/PhotoCamera';
+import SendIcon from '@mui/icons-material/Send';
+import SettingsIcon from '@mui/icons-material/Settings';
+import StarsIcon from '@mui/icons-material/Stars';
+import Switch from '@mui/material/Switch';
+import Typography from '@mui/material/Typography';
 import useStyles from './SearchChannel.styles';
 import './styles.css';
 
 const ChannelList = () => {
   const [drawer, setDrawer] = useState(false);
+  const [isShown, setIsShown] = useState(false);
+
   const styles = useStyles();
 
   return (
     <>
       <Drawer anchor="left" open={drawer} onClose={() => setDrawer(false)}>
```

Comment:
We should take the drawer out of the SearchChannel component, they have two different responsibilities, and we should keep our components as much cleaner as possible.